events/mbed_lib.json

```diff
@@ -21,6 +21,7 @@
         "shared-highprio-eventsize": {
             "help": "Event buffer size (bytes) for shared high-priority event queue",
             "value": 256
-        }
+        },
+        "use-lowpower-timer-ticker": 0
```

Comment:
Could you expand this to have a "help" field as well. Something like:
"Enable use of low power timer and ticker classes. May reduce the accuracy of the event queue."